thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -9,81 +9,70 @@
       <a class="te-pill-selectors__side-link" {{action "onClickTuneSensitivity" this}}>Customize sensitivity</a>
       <div class="te-horizontal-cards__container">
         {{!-- Alert anomaly stats cards --}}
-        {{#each anomalyStats as |card|}}
-          <ul class="te-horizontal-cards__card">
-
-            <li class="te-horizontal-cards">
-              <div class="te-horizontal-cards__card-title">{{card.title}}</div>
-              {{#if card.tooltip}}
-                <div class="te-horizontal-cards__card-tooltip">
-                  <span>
-                    <i class="glyphicon glyphicon-question-sign"></i>
-                    {{#tooltip-on-element}}{{card.text}}{{/tooltip-on-element}}
-                  </span>
-                </div>
-              {{else}}
-                <div class="te-horizontal-cards__card-text">{{card.text}}</div>
-              {{/if}}
-            </li>
-
-            <li class="te-horizontal-cards__card-value te-horizontal-cards__card-value--tight">
-              <div class="te-horizontal-cards__val-group">
-                <div class="te-horizontal-cards__card-number">
-                  {{card.value}}
-                  {{#if (and card.units (not-eq card.value "N/A"))}}
-                    <span class="te-horizontal-cards__card-units">{{card.units}}</span>
-                  {{/if}}
-                </div>
-                <div class="te-horizontal-cards__card-subt {{if (eq card.projected "none") "te-horizontal-cards__card-subt--hide"}}">Projected:
-                  <strong>{{card.projected}}</strong>
-                  {{#if (and card.units (not-eq card.projected "N/A"))}}
-                    <span class="te-horizontal-cards__card-units--small">{{card.units}}</span>
-                  {{/if}}
-                </div>
-              </div>
-            </li>
-
-          </ul>
-        {{/each}}
+        {{anomaly-stats-block
+          isTunePreviewActive=isTunePreviewActive
+          displayMode="explore"
+          anomalyStats=anomalyStats
+        }}
       </div>
     </div>
 
+    {{#if isReportSuccess}}
+      {{#bs-alert type="success" class="te-form__banner te-form__banner--success"}}
+        <strong>Success:</strong> We have saved the reported anomaly for alert id <strong>{{alertId}}</strong>
+      {{/bs-alert}}
+    {{/if}}
+
+    {{#if isReportFailure}}
+      {{#bs-alert type="danger" class="te-form__banner te-form__banner--failure"}}
+        <strong>Error:</strong> Failed to save reported anomaly.
+      {{/bs-alert}}
+    {{/if}}
+
     <div class="te-content-block">
       <h4>Anomalies over time ({{filteredAnomalies.length}})</h4>
       <a class="te-pill-selectors__side-link te-pill-selectors__side-link--high" {{action "onClickReportAnomaly" this}}>Report missing anomaly</a>
 
       <div class="te-pill-selectors">
         {{!-- Dimension selector --}}
-        <div class="te-pill-selectors__range-picker col-md-3">
+        <div class="te-form__select--wider col-md-3">
```

Comment:
fix correct bem usage would be `class="te-form__select te-form__select--wider`

Reply:
done